src/client/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx

```diff
@@ -1,6 +1,5 @@
 import styled from '@emotion/styled'
```

Comment:
There's a `CampaignCodeModal` under `Offer` folder. Should we change there as well?

Reply:
It's already implemented there, no? In this function: `handleAddCampaignCode`